_posts/2016-12-06-future.md

```diff
@@ -0,0 +1,444 @@
+
+When 6to5 became Babel as mentioned in [Not Born to Die](https://babeljs.io/blog/2015/02/15/not-born-to-die), it became a platform: a suite of tools designed to create the next generation of JavaScript tooling. No longer did it just compile ES6 to ES5, it allowed developers to build tools on top of it.
+
+Some other milestones:
+
+- In [5.0.0](https://babeljs.io/blog/2015/03/31/5.0.0), Babel aligned more with the [TC39 process](https://tc39.github.io/process-document/) by introducing `stages`, added a `.babelrc` config option, and created a plugin system for custom transforms.
+- In [6.0.0](https://babeljs.io/blog/2015/10/29/6.0.0), Babel became modular (a pretty controversial idea at the time). This was a huge change that led to opt-in functionality (no defaults) and the concept of `Presets` and Plugin Options.
+- [6.1.0](https://github.com/babel/babel/releases/tag/v6.13.0) finally added [Preset Options](http://babeljs.io/docs/plugins/#plugin-preset-options).
+- [6.3.13](https://github.com/babel/babel/releases/tag/v6.13.3) Sebastian extracted our [monorepo](https://github.com/babel/babel/blob/master/doc/design/monorepo.md) build/publish tools into what is now [Lerna](https://github.com/lerna/lerna). (What's funny is [James](https://github.com/thejameskyle) rewrote it 3 times and I had to review everything)
+- [6.14.0](http://babeljs.io/blog/2016/08/24/6.14.0) added a [latest-preset](http://babeljs.io/docs/plugins/preset-latest/) that keeps up to date with the yearly JavaScript specification.
+- [6.16.0](http://babeljs.io/blog/2016/09/28/6.16.0) allowed changing out the parser or code-generator.
+- In August, we released a [minifier based on Babel](https://babeljs.io/blog/2016/08/30/babili).
+- In September, we released the first version of [babel-preset-env](https://github.com/babel/babel-preset-env).
+- After a year on Phabricator, we moved back to to GitHub issues thanks solely to [@danez](https://github.com/danez) and his amazing (and underappreciated) work.
+
+Now `babel-core` is downloaded over 5 million times amonth and almost 60 million times total.
+
+Thanks everyone for your continued support! We want to continue acting as the foundation of the JavaScript toolchain: compilation, linting, minification, codemods, code coverage, etc.
+
+## Current Status
+
+We understand that working on any of these could be its own project. If you're interested in helping out please check out the issues linked below!
+
+### Maintaining Babel plugins [for each proposal in TC39](https://github.com/tc39/proposals) starting from Stage 0
```

Comment:
This mentions TC39 without explaining what TC39 actually *is*, which is a bit confusing. I think most users of Babel wouldn't know what TC39 is.